src/suricata.c

```diff
@@ -1467,6 +1468,16 @@ static TmEcode ParseCommandLine(int argc, char** argv, SCInstance *suri)
                 AppLayerParserSetup();
                 RegisterModbusParsers();
                 exit(AppLayerParserFromFile(ALPROTO_MODBUS, optarg));
+            } else if(strcmp((long_opts[option_index]).name, "afl-enip-request") == 0) {
```

Comment:
Looks like you missed this comment: https://github.com/inliniac/suricata/pull/2012#discussion_r59835625

--afl-enip options are not functional currently